### PR TITLE
Initial draft of Spawning-15m dataset card.

### PR DESCRIPTION
**Pull request by Padge91**
Initial draft of the Spawning-15m card content.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: datasets/pd12m.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      AuthorName: Spawning
+    - Title: Downloading Images
+      URL: https://huggingface.co/datasets/Spawning/PD12M/blob/main/tutorials/images.md
+      AuthorName: Spawning
+  Tools & Applications:
+    - Title: Hugging Face Dataset
+      URL: https://huggingface.co/datasets/Spawning/PD12M
+      AuthorName: Spawning
+    - Title: Source.Plus
+      URL: https://source.plus/collections/pd12m-mxenifxs
```
Comment (kc00k) [inline datasets/pd12m.yaml:33]: @Padge91 this URL seems to be 404'ing for me. Can you double check this is correct?

Comment (Padge91) [reply datasets/pd12m.yaml]: @kc00k Apologies, this has been fixed in a new commit.

Comment (kc00k) [reply datasets/pd12m.yaml]: @Padge91 Thank you. Your branch is now out of date, once you merge main back into your branch I will move forward with merging your branch in.

Comment (Padge91) [reply datasets/pd12m.yaml]: @kc00k Done!

**Pull request merged**